benchexec/tools/frama-c-sv.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.util as util
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for Frama-C with a wrapper for usage in the SVCOMP.
+    URL: https://gitlab.com/sosy-lab/software/frama-c-sv
+    """
+
+    REQUIRED_PATHS = ["."]
+    _TOOL_NAME = "frama-c-sv"
+    _SCRIPT_NAME = _TOOL_NAME + ".py"
+
+    def executable(self, _):
+        return util.find_executable(_SCRIPT_NAME)
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
```

Comment:
This seems to contradict the `executable` method: That method does not search in a subdirectory, but `program_files` still marks everything in the parent directory to be required (which is typical if the executable is in a subdirectory like `bin`, but then `executable` should search there). If the executable is in the tool's main directory, just deleted this method, the default implementation would then work well.

Reply:
Sure, I will delete it then